File: snippets/js/array/move/move.js
var sourceArray = [5, 4, 3, 2, 1];
var destinationArray = ['a', 'b', 'c'];
var sourceIndex = 1;       // 0: first item, 1: second item...
var destinationIndex = 2;  // 2: third item...
var count = 3;
console.log(sourceArray);       // [ 5, 4, 3, 2, 1 ]
console.log(destinationArray);  // [ 'a', 'b', 'c' ]
destinationArray.splice(destinationIndex, 0, ...sourceArray.splice(sourceIndex, count));  // ≡
console.log(sourceArray);       // [ 5, 1 ]
console.log(destinationArray);  // [ 'a', 'b', 4, 3, 2, 'c' ]

File: snippets/py/array/move/move.py
sourceArray = [5, 4, 3, 2, 1]
destinationArray = ['a', 'b', 'c']
sourceStart = 1       # 0: first item, 1: second item...
destinationIndex = 2  # 2: third item...
count = 3
print(sourceArray)       # [ 5, 4, 3, 2, 1 ]
print(destinationArray)  # [ 'a', 'b', 'c' ]
destinationArray = destinationArray[0:destinationIndex] + sourceArray[sourceStart:(sourceStart + count)] + destinationArray[destinationIndex:]  # ≡
del sourceArray[sourceStart:(sourceStart + count)]                                                                                              # ≡
print(sourceArray)       # [ 5, 1 ]
print(destinationArray)  # [ 'a', 'b', 4, 3, 2, 'c' ]